src/dia/Cell.mjs

```diff
@@ -309,6 +313,23 @@ export const Cell = Backbone.Model.extend({
         return this;
     },
 
+    _sortCellsByZIndex: function(cells) {
+        cells.sort((a, b) => {
+            const zIndexes = {
+                a: a.get('z'),
```

Comment:
Note that `z` could be also undefined. The function should account for that case too (it's an equivalent of `0`).

Anyway please use the `util.sortBy` instead.